src/modules/ui.js

```diff
@@ -0,0 +1,164 @@
+
+    // Create div for edit view
+    const divEdit = document.createElement('div');
+    divEdit.className = 'editView hide';
+
+    const inputEdit = document.createElement('input');
+    inputEdit.setAttribute('type', 'text');
+    inputEdit.className = 'inputEdit txt-mPlus';
+    inputEdit.value = task.description;
+
+    divEdit.appendChild(inputEdit);
+
+    const aAccept = document.createElement('a');
+    aAccept.className = 'acceptIcn';
+    const iconAccept = document.createElement('i'); // creates accept icon
+    iconAccept.className = 'fas fa-check-circle txt-mPlus';
+    aAccept.appendChild(iconAccept); // appends accept icon to anchor
+    divEdit.appendChild(aAccept); // appends accpet anchor to item
+
+    const aRemove = document.createElement('a');
+    aRemove.className = 'removeIcn';
+    const iconRemove = document.createElement('i'); // creates remove icon
+    iconRemove.className = 'fas fa-trash txt-mPlus';
+    aRemove.appendChild(iconRemove); // appends remove icon to anchor
+    divEdit.appendChild(aRemove); // appends remove anchor to item
+
+    item.appendChild(divEdit);// -- appends divEdit to item
+
+    list.appendChild(item); // appends item to list
+  }
+
+  static addEmptyToDoMessage() {
+    const list = document.querySelector('#taskList');
+
+    const item = document.createElement('li'); // creates list item
+    item.className = 'appItem';
+
+    const p = document.createElement('p');
+    p.className = 'noToDos';
+    p.textContent = "No to-do's right now";
+
+    item.appendChild(p);
+    list.appendChild(item);
+  }
+
+  static displayTasks() {
+    const list = document.querySelector('#taskList');
+    list.innerHTML = '';
+
+    const todos = Store.getTasks();
+    if (todos.length === 0) {
+      this.addEmptyToDoMessage();
+    } else {
+      todos.forEach((task) => this.addTask2list(task));
+    }
+  }
+
+  static addTask(newDescrip) {
+    const todos = Store.getTasks();
+    const neo = new Task(newDescrip, todos.length + 1);
+    todos.push(neo);
+    // updates indexes
+    todos.forEach((todo, i) => { todo.index = i + 1; });
+    Store.setTasks(todos);
+    if (todos.length === 1) {
+      document.querySelector('#taskList').innerHTML = '';
+    }
+    // adds single task: neo
+    this.addTask2list(todos[todos.length - 1]);
+  }
+
+  static changeLiToEditMode(li) {
+    const lisChildren = li.children;
+    // change clases of divs
+    const normalView = lisChildren[0];
+    normalView.classList.add('hide');
+
+    const editView = lisChildren[1];
+    editView.classList.remove('hide');
+
+    // sets focus con the input to edit
+    const inputEdit = editView.querySelector('input');
+    inputEdit.id = 'inputEdit';
+    inputEdit.focus();
+  }
+
+  static removeTask(index, li) {
+    const todos = Store.getTasks();
+    todos.splice(index, 1);
+    // updates indexes
+    todos.forEach((todo, index) => { todo.index = index; });
+    Store.setTasks(todos);
```

Comment:
- [ ] Kindly correctly update the indexes when a task is deleted. Right now when I delete a task, the first index starts from 0 instead of 1. It would be great if you could fix this issue.
![Annotation 2022-03-25 063424](https://user-images.githubusercontent.com/85113567/160060764-22a169a2-b525-4d1b-affa-9303613313b7.jpg)

Reply:
Thank you for pointing this out... Index updated.. 

![image](https://user-images.githubusercontent.com/40554384/160157476-eee7ebcf-1e13-45c6-a69c-c795f5457f79.PNG)